api/api.go

```diff
@@ -29,7 +28,14 @@ func NewApiService(params ApiServiceParams) *ApiService {
 	if params.ListenAddr == "" {
 		params.ListenAddr = DefaultListenAddress
 	}
-	return &ApiService{params: params, router: buildRouter()}
+	database, err := db.New(params.DBConfig)
+	if err != nil {
+		panic(fmt.Sprintf("failed to instantiate db: %+v", err))
+	}
+	a := &ApiService{params: params, db: database}
```

Comment:
nit: any reason to not use more descriptive var names? I assume you like the short ones combined with typings?

Reply:
api is the package and I was lazy about name thinking, apiService would work well